sdk/search/arm-search/package.json

```diff
@@ -3,16 +3,16 @@
   "sdk-type": "mgmt",
   "author": "Microsoft Corporation",
   "description": "A generated SDK for SearchManagementClient.",
-  "version": "4.0.0-beta.2",
+  "version": "1.0.0-beta.1",
```

Comment:
Why is the version number going backwards? This is the reason for the changelog verification failure.

Reply:
yes, package version will be updated when running changelog-tool from js-sdk-release-tools, I forget to run this after re-gen this package, thanks for help